common.lic

```diff
@@ -224,6 +224,20 @@ module DRC
     bput("weather", inside_no_sky, inside_yes_sky, outside) != inside_no_sky
   end
 
+  # Returns true if at least one moon (e.g. katamba, yavash, xibar)
+  # is above the horizon and won't set for at least another ~4 minutes.
+  def moons_visible?
+    !visible_moons.empty?
+  end
+
+  # Returns list of moon names (e.g. katamba, yavash, xibar)
+  # that are above the horizon and won't set for at least another ~4 minutes.
+  def visible_moons
+    DRCA.check_moonwatch
```

Comment:
Does this introduce a circular dependency?

Reply:
I noticed there's no `custom_require.call(%w[common-arcana])` in this file, but DRCA requires DRC

Reply:
Should these be in DRCA or DRCMM?

Reply:
🤦‍♂️ Well that's embarrassing. Good catch on the circular dependency, and yes I agree these should move to common-moonmage (as moon mages are the primary use case for these methods)

Reply:
I've moved the new methods to `common-arcana` alongside the `check_moonwatch` function they depend on